pennylane/templates/layers.py

```diff
@@ -111,16 +111,16 @@ def _cv_neural_net_layer(theta_1, phi_1, varphi_1, r, phi_r, theta_2, phi_2, var
 
 
 def StronglyEntanglingLayers(weights, wires, ranges=None, imprimitive=CNOT):
-    r"""A sequence of layers specified in `arXiv:1804.00633 <https://arxiv.org/abs/1804.00633>`_. The layers are
-    designed to rapidly entangle the qubits and consist of different.
-    The 2-qubit gates, whose type is specified by the ``imprimitive`` argument,
-    act chronologically on the wires with their first qubit, while the second qubit is
-    determined by a hyperparameter :math:`r` called the *range* (which indicates how many wires in a
-    linear chain of qubits lie between the two). If the first qubit is :math:`i`, the second qubit is
-    :math:`(i+r)\mod n`, where :math:`n` is the total number of wires in the template.
+    r"""Layers consisting of single qubit rotations and entanglers, inspired by the circuit-centric classifier design
+    `arXiv:1804.00633 <https://arxiv.org/abs/1804.00633>`_.
 
-    The number of layers :math:`L` is determined by the first dimension of ``weights``. The template is applied to
-    the qubits specified by the sequence ``wires``.
+    The argument ``weights`` contains the weights for each layer. The number of layers :math:`L` is therefore derived
+    from the first dimension of ``weights``.
+
+    The 2-qubit gates, whose type is specified by the ``imprimitive`` argument,
+    act chronologically on the :math:`M` wires, :math:`i = 1,...,M`. The second qubit of each gate is given by
+    :math:`(i+r)\mod M`, where :math:`r` is a  hyperparameter called the *range*, and :math:`r < M`.
+    If applied to one qubit only, this template will use no imprimitive gates.
```

Comment:
I realized that 0<r<M is important, else the imprimitive will attempt to act on one gate. Amazing what good tests reveal...